src/__init__.py

```
@@ -0,0 +1,98 @@
+            # Yield underpromotion moves
+            if not can_be_pawn_promotion(from_square, to_square):
+                continue
+            for piece_type in non_queen_promotion_piece_types:
+                promotion_move = copy(move)
+                promotion_move.promotion = piece_type
+                yield promotion_move
+
+
+def export(path: str | Path | PathLike, *, allow_overwrite: bool = False) -> None:
+    if os.path.exists(path) and not allow_overwrite:
+        raise FileExistsError(f"File {path} already exists")
```

Ruff (0.13.3)

Avoid specifying long messages outside the exception class

(TRY003)